Spaceport Culture (#5437)
endless-sky/endless-sky: data/human/culture republic.txt

#	minor
#	source
#		government "placeholder"
#		attributes "placeholder"
#		not attributes "station"
#	to offer
#		random < 1
#	on offer
#		conversation
#			`placeholder`
#				decline

#############################################
#      Republic with Syndicate Pairing      #
#############################################

mission "ZZCU Sad Gamblers -R"
	minor
	source
		government "Republic"
		attributes "near earth"
		not attributes "station"
	to offer
		random < 1
	on offer
		conversation
			`Passing through the spaceport you see a sizable group of people waiting on a lottery draw. You hear a whoop of success and your attention is drawn as one of them wins a moderate sum in a 12th division win; a day or two worth of standard pay.`
			`	It's difficult not to pity them as they immediately buy more tickets with the whole value.`
				decline

#############################################
#               Republic Only               #
#############################################

mission "ZZCU Gift Store Interaction"
	minor
	source
		government "Republic"
		attributes "deep"
		not attributes "station"
	to offer
		random < 1
	on offer
		conversation

> 			`As you wander the spaceport with a snack, waiting for your ship to be refueled, you pass a local souvenir shop and poke your head in to kill a little time. You see a decent looking mug with the text "<source>" printed underneath an attractive, stylized representation of the spaceport you are presently standing in and pick it up to see how it feels. The handle is solid, and not too small for your fingers, and the lip of it looks like it would be comfortable to drink out of.`

Author: See in this case it's a souvenir, so the planet would seem logical to me.

				decline
			label ignore
			`	You leave the mysterious smell to the locals and continue on your business.`
				decline

mission "ZZCU Paradise Cleaner Incident"
	minor
	source
		government "Republic"
		attributes "paradise"
		not attributes "station"
	to offer
		random < 1
	on offer
		conversation
			`Visiting the Paradise spaceports is always an interesting experience for someone from abroad. Nowhere else is class such an ingrained component of daily life. You find yourself noticing people who seem to be invisible to most of those going about their business, but they never catch your eye. They always appear to be doing their utmost to avoid notice.`
			`	You haven't noticed children that look poor here in the past, but on this occasion you spot one right before he knocks over a mop and bucket... right in front of a businessman and his family rolling their bags through the spaceport. You've never seen a mother go so deathly pale as the cleaner who scurried to fix things before the man had even drawn breath.`

> 			`	A pair of security officers appear almost immediately, before you could intercede one way or the other, as the man and his family begin to complain. One officer provides assurances as the other supervises the cleaner, who quickly hurries her children out of harms way.`

Reviewer: ```suggestion
			`	A pair of security officers appear almost immediately, before you could intercede one way or the other, as the man and his family begin to complain. One officer provides assurances as the other supervises the cleaner, who quickly hurries her child out of harms way.`
```
She isn't really mentioned as having more than one child before, so it feels a little weird to bring it up here. You could also keep this as-is and add some mention of her having additional kids earlier, but that might be more work.

# terms of the GNU General Public License as published by the Free Software
# Foundation, either version 3 of the License, or (at your option) any later version.
#
# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
# PARTICULAR PURPOSE.  See the GNU General Public License for more details.

# "Missions" which add culture events to the spaceport.

#mission "ZZCU cultural holiday template"
#	minor
#	source
#		government "placeholder"
#		attributes "placeholder"
#		not attributes "station"
#	to offer
#		and
#			month == placeholder
#			day >= placeholder
#			day <= placeholder
#	on offer
#		conversation
#			`placeholder`
#				decline
			
#mission "ZZCU cultural random template"
#	minor
#	source
#		government "placeholder"
#		attributes "placeholder"
#		not attributes "station"
#	to offer
#		random == 1
#	on offer
#		conversation
#			`placeholder`
#				decline

#############################################
#      Republic with Syndicate Pairing      #
#############################################

> mission "ZZCU Sad Gamblers -R"

Reviewer 2: You basically have this same thing in two different regions. Maybe have some difference between the regions? I'd also suggest changing the names. Actually type out "Republic" and "Syndicate" instead of "-R" and "-S".

Author: I was trying to get it to appear anywhere in either region but I couldn't get the source criteria to work right.

	minor
	source
		government "Republic"
		attributes "near earth"
		not attributes "station"
	to offer
		random == 1
	on offer
		conversation

> 			`Passing through the spaceport you see a sizable group of people waiting on a lottery draw. You hear a whoop of success and your attention is drawn as one of them wins a moderate sum in an 12th division win (a day or two worth of standard pay).`

Reviewer 3: a* 12th division win
Replace (a day or two worth of standard pay) with "which corresponds to a day or two of standard pay".

Reviewer 4: Quotations would imply that someone spoke them? I think it would be more fitting to say `a moderate sum in an 12th division win; a day or two worth of standard pay.`

Author: That's what I get for changing it from an 'eighth' division win without proofreading.
Thanks vkr, went with that.

#############################################

mission "ZZCU Gift Store Interaction"
	minor
	source
		government "Republic"
		attributes "deep"
		not attributes "station"
	to offer
		random == 1
	on offer
		conversation
			`As you wander the spaceport with a snack, waiting for your ship to be refueled, you pass a local souvenir shop and poke your head in to kill a little time. You see a decent looking mug with the text "<source>" printed underneath an attractive, stylized representation of the spaceport you are presently standing in and pick it up to see how it feels. The handle is solid, and not too small for your fingers, and the lip of it looks like it would be comfortable to drink out of.`
			`	Two young adults wander into the aisle just a couple of meters away. They're clearly from abroad judging by their conversation occurring in some local dialect which is neither local nor one you understand. They're cooing over an item they've found but when one flips it over to look at the price tag there is an immediate exchange of looks and expressions that does not require translation. The item ends up back on the shelf with a giggle and a head-shake.`

> 			`	Curious you turn over your own mug and see that it is indeed at least five times what you would expect to pay for a mug, even a good one. Still, t is certainly a novelty...`

Reviewer 4: `	Curious you turn` should be `	Curious, you turn`
`Still, t is certainly a novelty...` should be `Still, it is certainly a novelty...`

			choice
				`	(Buy the mug.)`
					goto buy
				`	(Put it back.)`
			`	You roll your eyes and put it back. Some experiences really are universal.`
				decline
			label buy

> 			`	You purchase the mug on an impulse. At least you can ditch your other one with a chip in it.`

Reviewer 2: Let's give the player a mug outfit and take 50 credits from them by making this an invisible mission where `on accept` you get the outfit and lose the credits, then automatically fail the mission.

"Mug: Hand to hand outfit. +0.1 boarding attack. You can throw it in a pinch, you guess." - @Galaucus

				decline

mission "ZZCU Trash Fire Crops"
	minor
	source
		government "Republic"
		attributes "dirt belt"
		not attributes "station"
	to offer
		random == 1
	on offer
		conversation

> 			`As you move about the spaceport you can't avoid the smell. It's like a trash fire, but nothing more than the smell is in the air.`

Reviewer 2: Took me like five tries to figure out what this was trying to say.
```suggestion
			`As you move about the spaceport you can't avoid the terrible smell. It's like a trash fire, but you can't see any sort of smoke.`
```

			choice
				`	(Ignore it.)`
					goto ignore
				`	(Ask around.)`

> 			`	You ask one of the locals who doesn't look too busy. They sadly inform you that there's been a bumper banana crop, but unfortunately other, wealthier, worlds had a bumper crop at the same time, and there's nowhere to either sell or store a significant excess. Distressed farmers are burning the excess fruit to make fertilizer for the next season, the wind has carried the smell all over town.`

Reviewer 2: ```suggestion
			`	You ask one of the locals who doesn't look too busy. They sadly inform you that there's been a bumper banana crop, but unfortunately other wealthier worlds had a bumper crop at the same time, and there's nowhere to either sell or store a significant excess. Distressed farmers are burning the excess fruit to make fertilizer for the next season, the wind has carried the smell all over town.`
```

				decline
			label ignore
			`	You leave the mysterious smell to the locals and continue on your business.`
				decline

mission "ZZCU Paradise Cleaner Incident"
	minor
	source
		government "Republic"
		attributes "paradise"
		not attributes "station"
	to offer
		random == 1
	on offer
		conversation

> 			`Visiting the Paradise spaceports is always an interesting experience for someone from abroad. Nowhere else is class such an ingrained component of daily life. You catch yourself noticing people who seem to be invisible to most of those going about their business, but they never catch your eye. Perhaps it is because of your background, but you notice them even when they are avoiding notice, and so you are able to notice something seconds before it happens.`

Reviewer 3: Repetition of "notice".
Also, are the "invisible" people easy to notice for first last or hard? It's really not clear.

Author: Removed one notice, but this circumstance is a little bit like that. Hopefully made it more clear.

> 			`Visiting the Paradise spaceports is always an interesting experience for someone from abroad. Nowhere else is class such an ingrained component of daily life. You catch yourself noticing people who seem to be invisible to most of those going about their business, but they never catch your eye. Perhaps it is because of your background, but you see them even when they are trying to avoid notice, and so you are able to notice something imminent seconds before it happens.`

Reviewer 2: This last sentence reads strange to me. Maybe just ditch it.

#	minor
#	source
#		government "placeholder"
#		attributes "placeholder"
#		not attributes "station"
#	to offer
#		random == 1
#	on offer
#		conversation
#			`placeholder`
#				fail

#############################################
#           Republic or Syndicate           #
#############################################

mission "Sad Gamblers"
	minor
	source
		or
			and
				government "republic"
				attributes "near earth"
			government "syndicate"
		not attributes "station"
	to offer
		random == 1
	on offer
		conversation
			`Passing through the spaceport you see a sizable group of people waiting on a lottery draw. You hear a whoop of success and your attention is drawn as one of them wins a moderate sum in an 12th division win (a day or two worth of standard pay).`
			`	It's difficult not to pity them as they immediately buy more tickets with the whole value.`
				fail

#############################################
#               Republic Only               #
#############################################

mission "Gift Store Interaction"
	minor
	source
		government "republic"
		attributes "deep"
		not attributes "station"
	to offer
		random == 1
	on offer
		conversation

> 			`As you wander the spaceport with a snack, waiting for your ship to be refuelled, you pass a local souvenir shop and poke your head in to kill a little time. You see a decent looking mug with the text "<source>" printed underneath an attractive, stylised representation of the spaceport you are presently standing in and pick it up to see how it feels. The handle is solid, and not too small for your fingers, and the lip of it looks like it would be comfortable to drink out of.`

Reviewer 5: refuelled => refueled

Author: Bloody american spellings. :P  Don't get too hung up on typos at this stage though, it hasn't been subjected to a proper spellcheck yet. (My initial ideas are on the wong computer.)

				fail
			label ignore
			`	You leave the mysterious smell to the locals and continue on your business.`
				fail

mission "Paradise Cleaner Incident"
	minor
	source
		government "republic"
		attributes "paradise"
		not attributes "station"
	to offer
		random == 1
	on offer
		conversation
			`Visiting the Paradise spaceports is always an interesting experience for someone from abroad. Nowhere else is class such an ingrained component of daily life. You catch yourself noticing people who seem to be invisible to most of those going about their business, but they never catch your eye. Perhaps it is because of your background, but you notice them even when they are avoiding notice, and so you are able to notice something seconds before it happens.`
			`	You haven't noticed children that look poor here in the past, but on this occasion you spot one right before he knocks over a mop and bucket... right in front of a businessman and his family rolling their bags through the spaceport. You've never seen a mother go so deathly pale as the cleaner who scurried to fix things before the man had even drawn breath.`

> 			`	A pair of security officers appear almost immediately, before you could intercede one way or the other, as the man and his family begin to complain. One officer provides assurances as the other supervises the cleaner, who quickly hurries her children out of harms way.`

Reviewer 5: is the cleaner the parent of the child or not?

Author: That is the intended implication.
